ASDKListKit/ASDKListKitTests/ASListKitTests.m

```diff
@@ -0,0 +1,107 @@
+//
+//  ASListKitTests.m
+//  AsyncDisplayKit
+//
+//  Created by Adlai Holler on 12/25/16.
+//  Copyright © 2016 Facebook. All rights reserved.
+//
+
+#import <XCTest/XCTest.h>
+#import <AsyncDisplayKit/AsyncDisplayKit.h>
+#import "ASListKitTestAdapterDataSource.h"
+#import "ASXCTExtensions.h"
+#import <JGMethodSwizzler/JGMethodSwizzler.h>
+
+@interface ASListKitTests : XCTestCase
+
+@property (nonatomic, strong) ASCollectionNode *collectionNode;
+@property (nonatomic, strong) UICollectionView *collectionView;
+@property (nonatomic, strong) IGListAdapter *adapter;
+@property (nonatomic, strong) ASListKitTestAdapterDataSource *dataSource;
+@property (nonatomic, strong) UICollectionViewFlowLayout *layout;
+@property (nonatomic, strong) UIWindow *window;
+@property (nonatomic) NSInteger reloadDataCount;
+
+@end
+
+@implementation ASListKitTests
+
+- (void)setUp {
```

Comment:
nit: The default-inserted methods use the brace-at-end-of-name style, so we should bump it down.